db/re/item_db_equip.yml

```
@@ -68177,6 +68222,62 @@ Body:
       bonus bMdef,5*(.@r/4);
       if (.@r>=12)
          bonus3 bAutoSpellWhenHit,"AL_PNEUMA",1,50;
+  - Id: 15363
+    AegisName: Mars_Armor_TW
+    Name: Mars Armor
+    Type: Armor
+    Weight: 300
+    Slots: 1
+    Jobs:
+      Assassin: true
+      Rogue: true
+    Classes:
+      All_Third: true
+    Locations:
+      Armor: true
+    ArmorLevel: 1
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r =  getrefine();
+      autobonus3 "{ bonus bAspd,2; }",1000,10000,"TF_BACKSLIDING","{ specialeffect2 EF_HASTEUP; }";
+      bonus bDex,3*(.@r/4);
+      bonus bMaxHPrate,3*(.@r/4);
+      bonus2 bIgnoreDefRaceRate,RC_DemiHuman,15;
+      bonus2 bIgnoreDefRaceRate,RC_Player_Human,15;
+      bonus2 bIgnoreDefRaceRate,RC_Brute,15;
+      bonus2 bIgnoreDefRaceRate,RC_Player_Doram,15;
+      bonus2 bSkillAtk,"SC_TRIANGLESHOT",2*(getskilllv("AC_DOUBLE"));
+      if (.@r>=12)
+         bonus2 bSkillCooldown,"GC_DARKCROW",-5000;
```

For me the skillatk bonus is in the +12 refine, the sentence being after the +12 refine check (like you did for the next item)
```suggestion
      if (.@r>=12) {
         bonus2 bSkillCooldown,"GC_DARKCROW",-5000;
         bonus2 bSkillAtk,"SC_TRIANGLESHOT",2*(getskilllv("AC_DOUBLE"));
      }
```

---

Thanks